versions/3.1.0.md

```diff
@@ -3162,6 +3163,87 @@ animals:
 </aliens>
 ```
 
+#### <a name="alternativeSchemaObject"></a>Alternative Schema Object
+
+This object makes it possible to reference an external file that contains a schema that does not follow the OAS specification. If tooling does not support the ``type``, tooling MUST consider the content valid but SHOULD provide a warning that the alternative schema was not processed.
+
+##### Fixed Fields
+Field Name | Type | Description
+---|:---:|---
+<a name="alternativeSchemaType"></a>type | `string` | **REQUIRED**. The value MUST match one of the values identified in the alternative Schema Registry.
+<a name="alternativeSchemaLocation"></a>location | `url` | **REQUIRED**.  This is a absolute or relative reference to an external resource containing a schema of a known type.  This reference may contain a fragment identifier to reference only a subset of an external document.
+
+This object MAY be extended with [Specification Extensions](#specificationExtensions).
+
+##### Examples
+
+Minimalist usage of alternative schema:
+
+```yaml
+schema:
+  alternativeSchema:
+    type: jsonSchema
+    location: ./real-jsonschema.json
+```
+
+Combination of OAS schema and alternative:
+
+```yaml
+schema:
+  type: object
+  nullable: true
+  alternativeSchema:
+    type: jsonSchema
+    location: ./real-jsonschema.json
+```
+
+Multiple different versions of alternative schema:
+
+```yaml
+schema:
+  anyOf:
+  - alternativeSchema:
+      type: jsonSchema
+      location: ./real-jsonschema-08.json
+  - alternativeSchema:
+      type: jsonSchema
+      location: ./real-jsonschema-07.json
+```
+
+Combined alternative schemas:
+
+```yaml
+schema:
+  allOf:
+  - alternativeSchema:
+      type: xmlSchema
+      location: ./xmlSchema.xsd
+  - alternativeSchema:
+      type: schematron
+      location: ./schema.sch
+```
+
+Mixed OAS schema and alternative schema:
+
+```yaml
+schema:
+  type: array
+  items:
+    alternativeSchema:
+        type: jsonSchema
+        location: ./real-jsonschema.json
+```
+
+##### Alternative Schema Registry
+
+It is important that there is a common understanding of the values which can be used to populate the alterative schema "type" property. An registry of alternative schema types is hosted at [https://spec.openapis.org/registries/alternative-schema](https://spec.openapis.org/registries/alternative-schema). This register serves as a controlled vocabulary which should be used to populate the "type" property.  The Inital contents of the registry include:
```

Comment:
```suggestion
It is important that there is a common understanding of the values which can be used to populate the alternative schema "type" property. A registry of alternative schema types is hosted at [https://spec.openapis.org/registries/alternative-schema](https://spec.openapis.org/registries/alternative-schema). This register serves as a controlled vocabulary which should be used to populate the "type" property.  The Initial contents of the registry include:
```

Reply:
Also "This register serves" -> "This registry serves"

Reply:
Note: link is dead for me

Reply:
@fenollp yeah, I think the expectation is that that registry will exist by the time OAS 3.1 is published, but it has not been set up yet.